composer/algorithms/gradient_clipping/gradient_clipping.py

```diff
@@ -133,8 +144,10 @@ def apply(self, event: Event, state: State, logger: Logger) -> Optional[int]:
 
         if event == Event.AFTER_TRAIN_BATCH and not state.deepspeed_enabled:
             apply_gradient_clipping(parameters=state.model.parameters(),
+                                    fsdp_modules=state.model.modules(),
```

Comment:
I wonder if this can be simplified to always just passing in `state.model` and just let `apply_gradient_clipping` do the logic of detecting and casing on FSDP modules.

Reply:
We can definitely let `apply_gradient_clipping` take care of detecting the FSDP modules! Was just wondering if there's an easier way to get to the modules that iterating through `.modules()`?